content/release-notes/2022-11-04.md

```diff
@@ -16,6 +16,11 @@ label: 'Console'
 
   _Neon Branching capabilities are not yet publicly available. If you would like to try this feature, contact us at  [iwantbranching@neon.tech](mailto:iwantbranching@neon.tech), describing your use case and requesting that Neon enable branching for your account._
 
+- UI: With the addition of support for new regions and updates to Neon's branching capabilities, changes were made to the hostname in Neon connection strings. Previously, a hostname had this format: `<project_id>.cloud.neon.tech`. With the introduction of new regions, a `<region_slug>` and `<platform>` value were added to the hostname for projects created in newly supported regions. With the update to branching capabilities, `<project_id>` was replaced by `<endpoint_id>`. As a result of these changes:
+  - Projects created in the original Neon region, US West (Oregon), have this hostname format: `<endpoint_id>.cloud.neon.tech`.
+  - Projects created in the newly supported regions, have this hostname format: `<endpoint_id>.<region_slug>.<platform>.neon.tech`.
```

Comment:
Just an idea, maybe some real example with changed a bit endpoint_id could be useful